### ISSUE TITLE
Issues encountered when upgrading from 2.3.0 to 3.0.0

### ISSUE THREAD

**Issue by duncanmak** [petitparser/dart-petitparser#75]
Hello, I just saw that 3.0.0 is out, congratulations!

I tried updating my `pubspec.yaml` to the new version and I see two breaking changes.

```
Error: The method 'anyIn' isn't defined for the class 'PSGrammarDefinition'.
Error: The method 'pick' isn't defined for the class 'Parser<dynamic>'.
```

1)  `PSGrammarDefinition` is derived from the lisp `grammar.dart` in the examples, so I was able to change `anyIn` to `anyOf` per https://github.com/petitparser/dart-petitparser/commit/287d88e8a1fe0777a721d03678813f6081cbc621#diff-4f3d5b971b9e0c8449d1a97a58a92e05

     It'd be nice to note that change in the ChangeLog.

2) I see that `pick` is now an extension method on `Parser`, but why is it that it says it's not defined?

I'm running Dart 2.7.1 and I kept the same import line as before:
```
import 'package:petitparser/petitparser.dart';
```

**Comment by renggli** [comment]
Thank you for reporting the issues, and sorry for making the migration unnecessary difficult.

I've added the following items to the `CHANGELOG.md` in d6c39dd38ad9712878d1cef2a44268496914b2e7:

  * `anyIn` has been removed in favor of the already existing and equivalent `anyOf` parser.
  * `pick` and `permute` are defined on `Parser<List>`, thus they won't be available on the more generic `Parser<dynamic>` any longer. Prefix the operators with a `castList` operator.
